basic/atm/ATM.py

```diff
@@ -0,0 +1,106 @@
+class ATM:
+    PATH_SAVING_FILE = "ATM_data_base.txt"
+
+    def __init__(self):
+        self._accounts_dict = {}
+        with open(ATM.PATH_SAVING_FILE, "r") as f:
+
+            account_lst = f.readlines()
+        for account in account_lst:
+            account = account.split()
+            self._accounts_dict[account[0]] = [account[1], int(account[2])]
```

Comment:
he meaning of each item of the list is not readable.
suggestion: use a dict with the keys "password" and "balance".